bittensor/synapses/gpt2.py

```diff
@@ -241,22 +237,24 @@ def forward(self,
         # local_hidden.shape = [batch_size, sequence_len, bittensor.__network_dim__]
         local_hidden = torch.cat([encoding, local_context], dim=2)
         local_hidden = self.hidden_layer(local_hidden)
-        output.local_hidden = local_hidden
+        
         if training:
             # local_target: projection of local_hidden onto target dimension.
             # local_target.shape = [batch_size, sequence_len, bittensor.__vocab_size__]
             local_target = self.target_layer(local_hidden)
-            output.local_target = local_target
+            #output.local_target = local_target
 
             # local_target_loss: MLM loss between local_target and passed targets.
             # local_target_loss.shape = [1]
             shift_logits = local_target[..., :-1, :].contiguous()
             shift_labels = inputs[..., 1:].contiguous()
             local_target_loss = self.loss_fct(
                 shift_logits.view(-1, shift_logits.size(-1)), shift_labels.view(-1))
-            output.local_target_loss = local_target_loss
             output.loss = output.loss + local_target_loss
-
+            output.local_target_loss = local_target_loss.item()
```

Comment:
if we take the item it means we are not training the local_target loss which means the distillation is not actually improving the model. Did you also take out the distillation loss? cause this will mean it does not distill